test/error.js

```diff
@@ -73,8 +73,8 @@ test('error.shortMessage does not contain stdout/stderr', async t => {
 });
 
 test('Original error.message is kept', async t => {
-	const {originalMessage} = await t.throwsAsync(execa('wrong command'));
-	t.is(originalMessage, 'spawn wrong command ENOENT');
+	const {originalMessage} = await t.throwsAsync(execa('noop', {cwd: 1}));
+	t.regex(originalMessage, /The "options.cwd" property must be of type string. Received type number/);
```

Comment:
This assertion might not need a regular expression.

```suggestion
	t.true(originalMessage.includes('The "options.cwd" property must be of type string');
```

Reply:
Changed as suggested